AerofoilSDL/ShaderCode/DrawQuad32P.cpp

```diff
@@ -28,8 +28,16 @@
 
 namespace GpBinarizedShaders
 {
+	//macos does not support precision in OpenGLES2
```

Comment:
Handle this by changing the precision defs in Functions.h to an empty string instead of changing every shader.